### ISSUE TITLE
[SCREENREADER]: Assistive devices are hearing too much information read when we populate search results

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#491]
## Issue
Focus is being moved from the map to the search results when keyboard users press an arrow key to shift the map in a direction, or when users press plus or minus to zoom the map. This was noted as an SC 4.1.1 violation. Screenshot attached below.

## Related Issues
* https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/490
* https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/492

## Audit Finding
* Note 8, Defect 3 of 3

## Acceptance Criteria
* As an assistive device user, I would like to hear the number of results that have changed instead of the entire string of results we are returning.

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/

## WCAG or Vendor Guidance (optional)
* [Parsing: Understanding SC 4.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)

## Screenshots or Trace Logs
![Screen Shot 2019-07-05 at 8.39.01 AM (3).png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/efc5c2dc-772d-4ec3-9cd3-1b75015452ac)

**Comment by 1Copenut** [comment]
Closing this as not a defect per guidance from VA 508 office.

/cc @Karak888
